tests/core/pyspec/eth2spec/test/merge/fork_choice/test_on_merge_block.py

```diff
@@ -45,7 +48,7 @@ def wrap(flag: AtomicBoolean):
     assert is_called.value
 
 
-@with_phases([MERGE])
+@with_merge_and_later
```

Comment:
I thought `test_on_merge_block.py` tests are MERGE-phase-only?
/cc @zilm13

Reply:
TBH, I'm not 100% sure on these, I'll revert the `test_on_merge_block.py` changes for now until there is clarity.